samples/client.c

```diff
@@ -526,6 +513,100 @@ access_region(int sock, int region, bool is_write, uint64_t offset,
     return 0;
 }
 
+static int
+set_migration_state(int sock, uint32_t state)
+{
+    static int msg_id = 0xfab1;
+    struct vfio_user_device_feature req = {
+        .argsz = 16,
```

Comment:
Why 16? Is this `sizeof struct vfio_user_device_feature + struct vfio_user_device_feature_mig_state`?

Reply:
Yes.

Reply:
OK then could you replace it?

Reply:
I have done 👍